packages/server/src/index.ts

```diff
@@ -22,7 +22,13 @@ import app from './server';
 import { setServerTimeout } from './koaJsonRpc/lib/utils'; // Import the 'setServerTimeout' function from the correct location
 
 async function main() {
-  const server = await app.listen({ port: process.env.SERVER_PORT || 7546 });
+  const port = process.env.SERVER_PORT || 7546;
+
+  // allow `host` to be undefined (same as 0.0.0.0) if not configured
```

Comment:
These comments shuld be part of the configuration md

Reply:
Ah yeah you're right it makes sense. Updated.